Library/Homebrew/rubocops/formula_desc_cop.rb

```diff
@@ -0,0 +1,75 @@
+require_relative "./extend/formula_cop"
+require_relative "../extend/string"
+
+module RuboCop
+  module Cop
+    module Homebrew
+      # This cop audits `desc` in Formulae
+      #
+      # - Checks for existence of `desc`
+      # - Checks if size of `desc` > 80
+      # - Checks if `desc` begins with an article
+      # - Checks for correct usage of `command-line` in `desc`
+      # - Checks if `desc` contains the formula name
+
+      class FormulaDesc < FormulaCop
+        def on_formula(node, class_node, _parent_class_node, body)
```

Comment:
I'm wondering if it's worth being called `audit_formula`; the `on_` naming scheme is better if people are familiar with RuboCop but I'd think the majority won't be.